Neo4j.Driver/Neo4j.Driver/IResultCursor.cs

```diff
@@ -60,14 +58,6 @@ public interface IResultCursor
         /// <returns>A task returning a <see cref="bool"/>. Task's result is true if there is any result to be consumed, false otherwise.</returns>
         Task<bool> FetchAsync();
 
-        /// <summary>
```

Comment:
Should this be removed form the interface? Is that not a breaking change to any customers that currently use it?

Reply:
The Current still exists but it is coming from the IAsyncEnumerator<IRecord>. I think we might want a new definition though as the documentation will be wrong for how fetch works?